src/optimize/base_optimizer.js

```diff
@@ -417,17 +423,6 @@ export default class BaseOptimizer {
       ],
     };
 
-    // We need to add react-addons (and a few other bits) for enzyme to work.
-    // https://github.com/airbnb/enzyme/blob/master/docs/guides/webpack.md
```

Comment:
As stated at this url, this was not longer necessary after upgrading to enzyme 3